### DISCUSSION TITLE
MobaXterm_Portable_v22.1_cn_v0.22-beta.1

### DISCUSSION THREAD

**Discussion by RipplePiam** [RipplePiam/MobaXterm-Chinese-Simplified#6] [General]
### 🎉🎉🎉 MobaXterm Portable v22.1 (2022-06-26) 汉化版发布啦 !
更新日志：
- 调整页面布局；
- 汉化设置》工具栏》按钮
- 修复 Bug #5，效果见附图，现可进行调整按钮顺序、增减按钮等操作

<hr /><em>This discussion was created from the release <a href='https://github.com/RipplePiam/MobaXterm-Chinese-Simplified/releases/tag/'>MobaXterm_Portable_v22.1_cn_v0.22-beta.1</a>.</em>

**Comment by Gold249** [comment]
win11家庭版无法打开
